helpdesk_mgmt_crm/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+helpdesk_mgmt_crm.access_helpdesk_ticket_create_lead,access_helpdesk_ticket_create_lead,helpdesk_mgmt_crm.model_helpdesk_ticket_create_lead,base.group_user,1,0,0,0
+helpdesk_mgmt_crm_salesman.access_helpdesk_ticket_create_lead,access_helpdesk_ticket_create_lead,helpdesk_mgmt_crm.model_helpdesk_ticket_create_lead,sales_team.group_sale_salesman,1,1,1,1
```

Comment:
Don't give by default read permissions to internal users, and unlink permission is not usually granted:

```suggestion
helpdesk_mgmt_crm_salesman.access_helpdesk_ticket_create_lead,access_helpdesk_ticket_create_lead,helpdesk_mgmt_crm.model_helpdesk_ticket_create_lead,sales_team.group_sale_salesman,1,1,1,0
```